app/helpers/home_helper.rb

```diff
@@ -0,0 +1,5 @@
+module HomeHelper
+  def signed_in?
+    session[:user] && !session[:user].empty?
+  end
+end
```

Comment:
Not necessary, since it's a [`helper_method`](http://api.rubyonrails.org/classes/AbstractController/Helpers/ClassMethods.html#method-i-helper_method) from the `ApplicationController`.